src/utils/inference.py

```diff
@@ -24,6 +24,8 @@
 try:
     height_model = load_model(
         '/app/models/height/outputs/best_model.ckpt/', compile=False)
+    MCNN_height_model = load_model(
```

Comment:
add other try except block for loading this model
or may be 1 function to load models which handle all exceptions(reduce code duplication)